server/client_manager.py

```diff
@@ -758,6 +759,9 @@ def change_area(self, area, password=""):
                 raise ClientError(
                     f"Failed to enter [{area.id}] {area.name}: User already in specified area."
                 )
+            # If they're forced to follow, no escape.
+            if self.forced_to_follow and self.following is not None and self.following.area != area:
+                raise ClientError("You can't escape when you've been forced to follow someone!")
```

Comment:
You should put this line into the "if not allowed" clause, that way mods, area owners, spectators can break the force follow, and you can also break the force follow if you enter the default area of a hub (meaning you can hop between hubs without being restricted needlessly).

Reply:
At line 772

Reply:
Wouldn't this just allow users to escape by going to the default area of the hub they're currently in? I'll commit the change but that would kinda defeat the purpose if there's such an easy loophole.

Reply:
> Wouldn't this just allow users to escape by going to the default area of the hub they're currently in? I'll commit the change but that would kinda defeat the purpose if there's such an easy loophole.

Anyone can reach area 0 anyway, plus the affected user can just disconnect and reconnect to bypass the command